inst/eDISH_app/ui.R

```diff
@@ -0,0 +1,38 @@
+library(shiny)
+
+navbarPage("eDISH Shiny app", id = "inTabset",
```

Comment:
Suggest adding a "Readme" or "Wiki" tab in the Shiny app to guide users.

Reply:
filed #59